### PR TITLE
Version 2.3.0-rc1 release

### PR DESCRIPTION
**Pull request by harsharahul**
### Changed
* The SDK now supports version 20.1.00 of the DocuSign eSignature API.
* SDK Release Version updated.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,31 +12,19 @@
...
 #
 # python setup.py install
 #
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-
 REQUIRES = ["urllib3 >= 1.15", "six >= 1.8.0", "certifi >= 14.05.14", "python-dateutil >= 2.5.3", "setuptools >= 21.0.0", "PyJWT>=1.7.1", "cryptography>=2.5", "nose>=1.3.7"]
 
-class CleanCommand(Command):
```
Comment (gsnavin) [inline setup.py]: @harsharahul what are these changes? Don't we need this anymore?

Comment (harsharahul) [reply setup.py]: yup... @mmallis87  had added these changes along with updates to swagger-codegen-ignore file as these were not necessary. Brought in those changes.

Comment (gsnavin) [reply setup.py]: did you run the tests locally?

Comment (harsharahul) [reply setup.py]: yup, did and passed.

**Pull request merged**